index.html

```diff
@@ -648,6 +653,28 @@
 &lt;/script&gt;
 </pre>
       </section>
+      <section>
+        <h3>
+          Passing locale information with a message
+        </h3>
+        <pre class="example highlight">
+&lt;!-- controller.html --&gt;
+&lt;script&gt;
+  connection.send("{string: 'Hello, world!', lang: 'en-US'}");
```

Comment:
+1 to include a non-ASCII example. I don't think we need to bother with the `switch` clause to make our point, though:

```javascript
connection.send("{string: '你好，世界!', lang: 'zh-CN'}");
connection.send("{string: 'こんにちは、世界!', lang: 'ja'}");
connection.send("{string: '안녕하세요, 세계!', lang: 'ko'}");
connection.send("{string: 'Hello, world!', lang: 'en-US'}");
```